eclass/sgml-catalog.eclass

```diff
@@ -8,7 +8,10 @@
 # Author Matthew Turk <satai@gentoo.org>
 # @BLURB: Functions for installing SGML catalogs
 
-inherit base
+case ${EAPI:-0} in
+	1|2|3|4|5) inherit base ;;
```

Comment:
What about EAPI=0?

Reply:
I thought EAPI=0 is only implicitly assumed whenever EAPI is missing. I can add it.
Does it need some special care,, I.e. base eclass was not present in EAPI=0 or something?